closure/goog/editor/plugins/enterhandler.js

```diff
@@ -245,7 +245,7 @@ goog.editor.plugins.EnterHandler.prototype.deleteBrGecko = function(e) {
         //    end of that.
         // 2. If the BR doesn't have a previous sibling or the previous sibling
         //    is a block level element or a BR, we place the cursor at the
-        //    beginning of the leftmost leaf of its next sibling.
+        //    leftmost of the leftmost leaf of its next sibling.
```

Comment:
I don't think this should have changed - this comment seems accurate as it was.

Reply:
Thanks!!

d0cec01